### PR TITLE
Singularise unsent message prompt, if applicable

### PR DESCRIPTION
**Pull request by turt2live**
Adds vector-im/riot-web#1217

### CODE REVIEW

**Comment by pafcu** [comment]
Why is this not using the built-in counterpart functionality rather than checking the numbers explicitly?

**Comment by turt2live** [comment]
What built-in functionality?

**Comment by pafcu** [comment]
Seems like it is not documented anywhere. See e.g.

https://github.com/matrix-org/matrix-react-sdk/blob/6c5f4983daefcb9f3fcc196a6158a4d9d0588c6f/src/components/structures/RoomStatusBar.js#L278
and 
https://github.com/matrix-org/matrix-react-sdk/blob/6c5f4983daefcb9f3fcc196a6158a4d9d0588c6f/src/i18n/strings/en_EN.json#L100
https://github.com/matrix-org/matrix-react-sdk/blob/6c5f4983daefcb9f3fcc196a6158a4d9d0588c6f/src/i18n/strings/en_EN.json#L101

Basically, pass in the count to `_t()` and mark the singular version with "|one" and the plural with "|other" in the string JSON files.

**Comment by turt2live** [comment]
Interesting, I didn't know that existed. I'll test it and get the PR converted over to that.

**Comment by turt2live** [comment]
I took a look, and it looks like `_tJsx` doesn't support that functionality. Because 50% of this PR is using tJsx, I kinda feel it's fine to do the manual check in both places.

**Comment by t3chguy** [comment]
this is a functionality of counterpart so should be inherited by both _t and _tJsx

**Comment by turt2live** [comment]
_tJsx doesn't support a way to pass the count in, and I'm not about to try and touch the translation code for fear of breaking it.

**Comment by t3chguy** [comment]
Oh true, https://github.com/matrix-org/matrix-react-sdk/blob/master/src/languageHandler.js#L106 could be edited to pass through a 4th opt-arg from _tJsx

**Comment by turt2live** [comment]
Well, if it is that simple then I might as well. I'll poke it a bit later and see what happens.

**Comment by t3chguy** [comment]
Changing https://github.com/matrix-org/matrix-react-sdk/blob/master/src/languageHandler.js#L84
to `export function _tJsx(jsxText, patterns, subs, args) {` and then https://github.com/matrix-org/matrix-react-sdk/blob/master/src/languageHandler.js#L106 to `const tJsxText = _t(jsxText, args);` should be all it needs

**Comment by pafcu** [comment]
@turt2live `_tJsx()` is no more, so this needs to be ported over to use `_t()` instead. On the plus side, it allows you to pass the `count` variable, as discussed above.

**Comment by turt2live** [comment]
I'm not really able to maintain this PR. Someone else is more than welcome to do so.

**Comment by t3chguy** [comment]
I shall take it

**Comment by t3chguy** [comment]
Revived in https://github.com/matrix-org/matrix-react-sdk/pull/1692 due to no permission to original branch
